src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.MoveTypeCodeAction.cs

```diff
@@ -36,12 +36,14 @@ private string CreateDisplayText()
             {
                 switch (_operationKind)
                 {
-                    case OperationKind.MoveType:
+                    case MoveTypeOperationKind.MoveType:
                         return string.Format(FeaturesResources.Move_type_to_0, _fileName);
-                    case OperationKind.RenameType:
+                    case MoveTypeOperationKind.RenameType:
                         return string.Format(FeaturesResources.Rename_type_to_0, _state.DocumentNameWithoutExtension);
-                    case OperationKind.RenameFile:
+                    case MoveTypeOperationKind.RenameFile:
                         return string.Format(FeaturesResources.Rename_file_to_0, _fileName);
+                    case MoveTypeOperationKind.MoveTypeNamespaceScope:
+                        return string.Empty;
```

Comment:
?

Reply:
There's no display text associated with this right now, but that's intended. If we added a code fix provider for this type of change, we would add a string here.